### PR TITLE
Add news items

### PR DESCRIPTION
**Pull request by adrianhopebailie**
Link to Chain ref of ILP
Link to Conditional Payments blog
Fix formatting of news page

### CODE REVIEW

File: news.html [hunk 1]
```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
```
Comment (emschwartz) [inline news.html]: Instead of making the section wider, I think we should just make this particular bit of text shorter

File: news.html [hunk 2]
```diff
@@ -8,13 +8,21 @@
...
           <h1 class="page-header">Interledger News</h1>
 
           <p class="intro">Here you can find the latest Interledger developments and community announcements.</p>
 
 
          <ul class="list-group">
+         <li class="list-group-item">
+             <span class="label label-default label-pill pull-right">Oct 24</span>
+             Chain releases their <a target="_blank" href="https://chain.com/docs/protocol/papers/whitepaper#9-interoperability">Chain Protocol Whitepaper</a> citing Interledger for cross-chain interoperability.
```
Comment (emschwartz) [inline news.html]: How about just "Chain Protocol Whitepaper cites Interledger for cross-chain interoperability"?

File: news.html (hunk 1 above)
Comment (emschwartz) [inline news.html]: It makes the column smaller, which I think looks better

Comment (adrianhopebailie) [inline news.html]: I disagree. It makes that table float and look like there is something missing on the page.

The way bootstrap works is you use CSS classes to build a grid layout. Giving something a class of `col-sm-8` implies you'll put a column with `col-sm-4` or similar before or after.

If you look at `index.html` you can see how the first column is a 4/12 and then the second is 8/12. This was mistakenly set at 8 and should be a 12.

Comment (bsteinlo) [inline news.html]: Please just put this back to what it was and we can merge.

Comment (adrianhopebailie) [inline news.html]: @bsteinlo this causes the page to render with a huge amount of dead space to the right which makes the it look like something is missing from the page. Is that intentional?

screen shot 2016-10-25 at 21 59 53

Comment (bsteinlo) [inline news.html]: It is intentional. It's for general readability and UX, as there is very little text on the page. Please keep as is.

**Pull request merged**